### ISSUE TITLE
Example app showcasing integration

### ISSUE THREAD

**Issue by Johennes** [jhugman/uniffi-bindgen-react-native#137]
@trevoranderson has recently reported issues getting a library built with UBRN integrated into an application. I think this process can be quite fiddly as we've seen with the example app bootstrapped by `create-react-native-library` in the latest version. Maybe it would be worth to actually create an example app integrating an example library generated with UBRN?

This could possibly also be helpful for testing changes to UBRN or also for documenting changes for UBRN consumers. I think React Native itself must have something similar sitting behind the diffs in the [migration guide](https://react-native-community.github.io/upgrade-helper/?from=0.74.2&to=0.76.0-rc.6).

**Comment by jhugman** [comment]
This may be in the wheelhouse of @boek.

**Comment by boek** [comment]
I'll pick this up!

**Comment by trevoranderson** [comment]
Yeah, this is a great way to help people get started. Being able to check out an example environment that matches the end state of the official guide saves time when starting anew because you don't have to follow the guide step-by-step again, and also helps diagnose environment issues (if the sample app doesn't work, something in the environment is wrong)

**Comment by matthiasgeihs** [comment]
any progress on this?
